components/sup/src/manager/mod.rs

```diff
@@ -81,7 +81,7 @@ pub use self::sys::Sys;
 use self::user_config_watcher::UserConfigWatcher;
 use super::feat;
 use census::{CensusRing, CensusRingProxy};
-use config::GossipListenAddr;
+use config::{EnvConfig, GossipListenAddr};
```

Comment:
```
error[E0252]: the name `EnvConfig` is defined multiple times                                                                                                                                                
  --> components/sup/src/manager/mod.rs:84:14                                                                                                                                                               
   |                                                                                                                                                                                                        
51 | use config::EnvConfig;                                                                                                                                                                                 
   |     ----------------- previous import of the trait `EnvConfig` here                                                                                                                                    
...                                                                                                                                                                                                         
84 | use config::{EnvConfig, GossipListenAddr};                                                                                                                                                             
   |              ^^^^^^^^^ `EnvConfig` reimported here                                                                                                                                                     
```

Reply:
This is a result of me rebasing against master.  I have this fixed in the changes I'm making based on @christophermaier 's feedback.